snap/export_test.go

```diff
@@ -20,6 +20,9 @@
 package snap
 
 var (
-	ImplicitSlotsForTests        = implicitSlots
-	ImplicitClassicSlotsForTests = implicitClassicSlots
+	ImplicitSlotsForTests         = implicitSlots
+	ImplicitClassicSlotsForTests  = implicitClassicSlots
+	RelativeHooksDir              = relativeHooksDir
+	AddImplicitHooks              = addImplicitHooks
```

Comment:
not sure there's a big win in term of testing of exposing these, instead of calling the Read* functions (that are actually what will be used and call them)